From 873ff6f07092ca6fdbd9443e02a2823d3e4bd012 Mon Sep 17 00:00:00 2001
From: Gergely Nagy <algernon@madhouse-project.org>
Date: Sat, 30 Jun 2012 13:06:11 +0200
Subject: [PATCH] Make Gimp produce better man page output

Patch by carol <cspears@cablespeed.com> 18 Feb 2003

With this patch applied to aalib, Gimp produces better output for:
  Save as, ASCII art (*.txt, *.ansi, *.text)
  Format: For including in a man page
  Save

Origin: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=181558
---
 src/aasave.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/aasave.c b/src/aasave.c
index e96f10e..ea57fbb 100644
--- a/src/aasave.c
+++ b/src/aasave.c
@@ -270,13 +270,13 @@ __AA_CONST struct aa_format aa_roff_format =
     NORMALPRINTS,
     {"",
      "",
-     "\n.B ",
+     "\\fB",
      "",
      ""
     },
     {"",
      "",
-     "\n",
+     "\\fP",
      "",
      ""
     },
-- 
1.7.10.4

